tools/tf2openapi/types/tfmetagraph.go

```diff
@@ -30,6 +31,16 @@ func NewTFMetaGraph(metaGraph *pb.MetaGraphDef) (TFMetaGraph, error) {
 	return tfMetaGraph, nil
 }
 
-func (t *TFMetaGraph) Schema() *openapi3.Schema {
-	return &openapi3.Schema{}
+func (t *TFMetaGraph) Schema(sigDefKey string) (*openapi3.Schema, error) {
+	for _, sigDef := range t.SignatureDefs {
+		if sigDefKey != sigDef.Key {
+			continue
+		}
+		schema, err := sigDef.Schema()
```

Comment:
Can you just return this directly? Best practice is to ignore the first return value if an err isn't null, so it should be fine to pass through.

In fact, all of this might be cleaner if you do:
```
for _, sigDef := range t.SignatureDefs {
		if sigDefKey == sigDef.Key {
			returnsigDef.Schema()
		}
}
```

Reply:
Yeah, that is so much cleaner! Making this change for tfsavedmodel.go as well